tests/helper/helper_generic.go

```diff
@@ -167,3 +168,13 @@ func GetUserHomeDir() string {
 	Expect(err).NotTo(HaveOccurred())
 	return homeDir
 }
+
+// LocalKubeconfigSet checks for the running CI and sets the KUBECONFIG to the temporary config directory respectively
+func LocalKubeconfigSet(context string) {
+	homeDir := GetUserHomeDir()
+	if os.Getenv("CI") == "openshift" {
```

Comment:
can you try something  like this?
```go
	originalKubeCfg := os.Getenv("KUBECONFIG")
	if originalKubeCfg == "" {
		originalKubeCfg = filepath.Join(homeDir, ".kube", "config")
	}
	copyKubeConfigFile(originalKubeCfg, filepath.Join(context, "config"))
```

Reply:
> can you try something like this?
> 
> ```go
> 	originalKubeCfg := os.Getenv("KUBECONFIG")
> 	if originalKubeCfg == "" {
> 		originalKubeCfg = filepath.Join(homeDir, ".kube", "config")
> 	}
> 	copyKubeConfigFile(originalKubeCfg, filepath.Join(context, "config"))
> ```

It is already there in the function https://github.com/openshift/odo/blob/99f1a2f84fef58c2aafbce9a9bc44e47f7e63fe8/pkg/util/util.go#L976 which I am using [here](https://github.com/openshift/odo/pull/3041/files#diff-318517042700080474192fc7dc04e456R33)